recipes/dbghaplo/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version="0.0.2" %}
+
+package:
+  name: dbghaplo
+  version: {{ version }}
+
+source:
+  url: https://github.com/bluenote-1577/dbghaplo/archive/v{{ version }}.tar.gz
+
+  sha256: e7e2741afb0c7f12718ec969815d3c8f18ce7ba66517e21c6fcfb3fe4262780b
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('dbghaplo', max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - rust >=1.70
+    - make
+    - cmake >=3.12
+  run:
+    - python 
+    - samtools
+    - minimap2
+    - lofreq >=2.1.5
+    - tabix
+    - pysam >=0.16
+
+test:
+  commands:
+    - dbghaplo -h
+    - samtools --help
+    - minimap2 --help
+    - lofreq version
+    - tabix --help
+    - run_dbghaplo_pipeline -h
+    - haplotag_bam -h
```

Comment:
I see there's also a `write_contig_headers_vcf.py` script in your repo too. Should that be tested for here too?

Reply:
I think it's okay to not test it. It doesn't have any dependencies except baseline python and I wasn't going to advertise its usage heavily, more of a just in case you need it sort of thing. Let me know if that sounds ok.

Reply:
Thanks for all the help @mbhall88 !!

Reply:
No worries. This is probably a good reminder for me to add a section to the contributing docs documenting all of this.